meson.build

```diff
@@ -8,7 +8,13 @@ project('MangoHud',
 
 cc = meson.get_compiler('c')
 cpp = meson.get_compiler('cpp')
-prog_python = import('python').find_installation('python3', modules: ['mako'])
+
+python_deps = ['mako']
+if get_option('mangoplot')
+  python_deps += ['matplotlib', 'PyQt5']
+endif
+
+prog_python = import('python').find_installation('python3', modules: python_deps)
```

Comment:
Is this check even necessary for building? Shebang in py runs first found python3 anyway.

Reply:
It's not, just for packagers to see the dependency x)

Reply:
Should be optional then

Reply:
I am not sure I understand with "should be optional", installing `mangoplot` is already optional (but default on) and the dependency is removed if installing `mangoplot` is disabled.

Reply:
I.e. `check_runtime_dependencies`. When building packages in distributions usually runtime deps are not installed to reduce package build times (relevant if you do massive rebuilds, period CI builds, etc)

Reply:
where is this `check_runtime_dependencies` ? x)

Reply:
No, you need to make it a build option yourself